clang/include/clang/Driver/Options.td

```diff
@@ -7057,6 +7057,8 @@ def fsycl_help_EQ : Joined<["-"], "fsycl-help=">, Flags<[NoXarchOption]>,
 def fsycl_help : Flag<["-"], "fsycl-help">, Alias<fsycl_help_EQ>,
   Flags<[NoXarchOption]>, AliasArgs<["all"]>,
   HelpText<"Emit help information from all of the offline compilation tools">;
+def fsycl_libdevice_path_EQ : Joined<["-"], "fsycl-libdevice-path=">,
```

Comment:
No, it wasn't. Tbf I wasn't really aware the working group existed, I went with `-fsycl-libdevice-path` because it's similar to `-fsycl-libspirv-path`, but I'm happy to change it as you deem necessary, is there any other reviewer we could ping in this PR to get their opinion? Or how should I go about submitting this to the Options Working Group?

Reply:
@PietroGhg, you can check here https://wiki.ith.intel.com/display/ITSCompilersDevOps/Compiler+Options+WG for more information.  This should get enough eyes on the new option name.